docs/validator/economics.md

```diff
@@ -89,45 +105,47 @@ NEAR validators are subject to market and game-theory dynamics, relating to:
 3. Transaction fees and inflation
 
 **Total of NEAR tokens at stake**
-NEAR mints 5% new tokens every year, of which 10% go to the treasury, and the rest is split between validators. Therefore, the initial supply of 1 billion NEAR will generate 45 million tokens (4.5% of 1 billion) in stake rewards in the first year.
+NEAR mints 5% new tokens every year, of which 10% go to the treasury, and the rest is split between validators. Therefore, the initial supply of 1 billion NEAR will generate roughly 45 million tokens (4.5% of 1 billion) in stake rewards in the first year.
```

Comment:
should we mention contract reward?

Reply:
Aren't they part of the transaction fees/already circulating tokens?